app/templates/base.html

```diff
@@ -0,0 +1,71 @@
+{% extends 'govuk_frontend_jinja/template.html' %}
+
+{% set assetPath = url_for('static', filename='').rstrip('/') %}
+
+{% block pageTitle %}DLUHC Funding Service Design Iteration 1{% endblock %}
+
+{% block head %}
```

Comment:
might be worth making this a component. This is unlikely to change and it will simplify this page

Reply:
Agreed, this will also be a excellent opportunity to begin logically structuring the project. Opinions on where our macros/components should live team? I’ll have a look at the flask standard and report back in  this thread.

Reply:
@ramsharma-prog Will be nice to establish a nice project structure, maybe have a look at some best practices for flask macros :)

Reply:
@SentientHousePlant  Will do